pandas/io/sas/sas.pyx

```diff
@@ -9,6 +9,8 @@ ctypedef signed long long   int64_t
 ctypedef unsigned char      uint8_t
 ctypedef unsigned short     uint16_t
 
+cdef object np_nan = np.nan
```

Comment:
Still needed? (np.nan is just used below)

Reply:
Good catch, it was meant to be used (the `np.nan` lookup is quite slow)